.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+name: Release
+on:
+  create
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./tools/github-actions/setup-deps
+
+      # lint
+      - run: make lint-deps
+      - run: make -k lint
+
+  release:
+    runs-on: ubuntu-latest
+    needs: [ lint ]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./tools/github-actions/setup-deps
+
+      # build
+      - run: make build-multiarch
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
+
+      # test
+      - run: make go.test.unit
+
+      # push
+      - name: Login to DockerHub
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - run: make image-release
```

Comment:
prefer lesser targets to remember, can we use
```
TAG=${VERSION} make image.push.multiarch
```
instead ?

Reply:
I am looking into this